src/components/calculator/Calculator.jsx

```diff
@@ -1,95 +1,97 @@
 import React, { useState } from 'react';
 import calculate from '../../logic/calculate';
+import './Calculator.css';
```

Comment:
- [ ] I got this error when I ran your app. This prevented the app from running. Kindly import the file using the correct file name i.e. lower case `calculator.css`

  ```
  import './calculator.css';
  ```

  ![image](https://github.com/lucy-sees/MathMystics/assets/36542986/dd8ef035-d1b8-4bb1-a05d-9638dd606c9b)

Reply:
The filename for the calculator css is 'Calculator.css'

Reply:
My path name is correct because I have named the calculator css as "Calculator.css". 

![Capture](https://github.com/lucy-sees/MathMystics/assets/100368317/ad4bfe01-a398-47c4-bcef-4b162e4f7617)
I am not getting the same error. Kindly review my work 

https://github.com/lucy-sees/MathMystics/assets/100368317/1f18c1fc-c573-4d93-8849-9a252cb1f865

Reply:
@lucy-sees that's interesting, the calculator css file appears as lower case file name in my machine, and as you can see I have made no changes to this file.

![image](https://github.com/lucy-sees/MathMystics/assets/36542986/9bc5077c-b989-4926-a404-de644bba3805)

I see there are two CSS files with the same name in the repo, but both of us have one version of each. In that case I would recommend you get rid of one of the CSS files.

![image](https://github.com/lucy-sees/MathMystics/assets/36542986/750f7bc3-4de8-488f-9a09-a75b78575579)